src/config.rs

```diff
@@ -288,6 +288,15 @@ pub struct SharedConfigValues {
     #[cfg_attr(not(feature = "dbus_mpris"), structopt(skip), serde(skip))]
     use_mpris: Option<bool>,
 
+    #[cfg_attr(
+        feature = "dbus_mpris",
+        structopt(long),
+        serde(alias = "unique-mpris", default)
+    )]
+    #[cfg_attr(not(feature = "dbus_mpris"), structopt(skip), serde(skip))]
+    unique_mpris: Option<bool>,
```

Comment:
Would be good to add some short documentation string above that will then appear in `spotifyd --help`.